src/Engine/ProtoCore/Parser/AssociativeAST.cs

```diff
@@ -1033,16 +1029,14 @@ public VarDeclNode(VarDeclNode rhs)
                 Name = rhs.ArgumentType.Name
             };
             NameNode = NodeUtils.Clone(rhs.NameNode);
-            access = rhs.access;
+            Access = rhs.Access;
             IsStatic = rhs.IsStatic;
             ExternalAttributes = rhs.ExternalAttributes;
         }
 
-        public List<AssociativeNode> Attributes { get; set; }
```

Comment:
Are we removing support of attribute on variables? Is it because we are not supporting class definition in DS now?